.github/workflows/release.yaml

```diff
@@ -0,0 +1,40 @@
+name: Release Helm Chart Package
+
+on:
+  push:
+    tags:
+      - "v*"
```

Comment:
For my understanding this workflow will run only when we create a tag starting with `v`?

Reply:
yes exactly!